app/controllers/draws_controller.rb

```diff
@@ -31,6 +33,11 @@ def destroy
     handle_action(**result)
   end
 
+  def start
```

Comment:
Agreed; especially since starting the draw could mean starting the actual draw / lottery phase (this was already bothering me). How about `initiate`? `open` doesn't really work for me - I get that the implication is "open for business," so to speak, but it feels like weird verb to use `DrawOpener` vs `DrawInitiator`. Thoughts?